### DISCUSSION TITLE
Cannot paste a message that contains newline characters

### DISCUSSION THREAD

**Discussion by trizen** [tutao/tutanota#6452] [Bugs]
- [ ] This is not a feature request (existing functionality does not work, **not** missing functionality).
 I will request features on [forum](https://www.reddit.com/r/tutanota/) or via support.
- [ ] I've searched and did not find a similar issue.

**Bug in mobile app**

**Describe the bug**
 Copying a message that contains newline characters, cannot be pasted into the Tutanota mail app.

**To Reproduce**
1. Write a freaking message inside your notes app, including some new lines
2. Copy the message
3. Try to paste the message inside a new mail
4. Enjoy the error of nothing happening

**Expected behavior**
Copy-pasting a message should freaking work, even if the messages includes newline characters.

**Smartphone (please complete the following information):**
 - Device: Moto E4
 - OS: Android
 - Version: 7.1.1

**Comment by rezbyte** [comment]
Hi @trizen ,
Sorry to hear you are having issues with pasting into the mail editor.

Did an error pop-up appear when you pasted the text? If so, please report the error by pressing the 'Report' button on the error pop up if you have not already.

What note taking app are you using?

Possibly related: #3945.

**Comment by trizen** [comment]
Any note taking app. The issue is the same. Even if I cut a part of the mesage from the Tutanote app, and trying to paste it somewhere else inside the message, if the copied message contains newline characters, the message cannot be pasted.

No error message is displayed. The paste option simply doesn't work (using the OpenBoard keyboard).

Why is this issue a discussion? Is the issue intentional? There is no plan to fix it?

**Comment by rezbyte** [comment]
No need to worry! The issue was turned into a discussion while we investigate what factors such as OS, keyboard etc. contributes to the issue. After that, we can work on a solution.

**Comment by wrdhub** [comment]
Okay I did some research and I believe I have a fix. see #6525  I agree that this one is super annoying. 

@trizen, if you don't mind answering one more question: does pasting work if you long press in the text field and select "paste" from there? 
Some keyboards display copied text at the top of the keyboard and if I tried selecting the text from there pasting would not work, but pasting from the long press did.

**Reply by trizen** [reply]
Thanks for looking into this. Pasting with long press then selecting "Paste" does seem to work. Only when selecting from the keyboard's clipboard doesn't work.

**Reply by wrdhub** [reply]
Good to hear that Pasting with long press works. The fix for pasting from clipboard should be out with the next release (don't know exactly when that will be, sometime in the next few weeks).
